src/index.less

```diff
@@ -1,13 +1,10 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@import "~nav-frontend-core/less/core";
+@import "~nav-frontend-typografi-style/src/index";
```

Comment:
For `core`, se annen kommentar.

For `typografi` så kan dere gjøre om på `.maincontent` til `.app` også slipper dere å importere den her. 
Ved bruk av `.app` settes også ett par andre css-attributter som kan være nyttige å ha; https://github.com/navikt/nav-frontend-moduler/blob/master/packages/node_modules/nav-frontend-typografi-style/src/index.less

Reply:
Hei, jeg fikset `.maincontent` til `.app` , men synes det blir dumt å fjerne less-importen siden stylingen vil ikke fungere dersom vi slutter å ta i bruk javascript-componentene fra typografi. Tenker derfor det er bedre å importere det en gang for mye.

Reply:
Hei,
De aller fleste komponenter i nav-frontend har en avhengighet til typografi, f.eks snakkeboble-komponenten. Så lenge en sånn komponent er i bruk vil dere få all stylingen. (det et ihvertfall tanken 😛,  vanskelig å verifisere på mobilen 😣).

Men om dere opplever problemer med det så kan den bare bli for nå, og evt fjernes ved en senere anledning om man finner ut at den ikke trengs da.

For om den blir med flere ganger så er ikke det krise, men det er såklart noe vi prøver å unngå når vi har mulighet.

Reply:
react-scripts skal fjerne duplisert kode når css koden blir generert. Så tror ikke det har så mye å si, men er det dumt å eksplisitt importere det ? 

Tenker det kanskje er litt dumt å ha regler som kommer inn implicit er der pga andre ting :P ?

Reply:
Ville dobbeltsjekket om du ikke er 100% sikker, for husker jeg måtte styre noe veldig med forken av CRA for å få den til å fjerne duplisert css. 😕 Men det var kanskje i dev-modus? 🤨

Når det kommer til implisitt vs eksplisitt så er hele nav-frontend designet rundt konseptet at man ikke skal trenge å håndtere less-import manuelt. 
Alternativet er å ha med en fil ala https://github.com/navikt/nav-frontend-moduler/blob/master/examples/legacy/src/index.less og selvom det er mulig, så ville jeg nok heller foretrekke å la webpack finne ut hva som bør inkluderes i bundelen.

Reply:
tja, har sjekket i chrome. f.eks denne appen
http://behovsvurdering.herokuapp.com/

Finner hver css regel bare 1 gang her. Mener når vi hadde problemet så, så vi samme regel duplisert flere ganger i chrome. Så tror react-scripts har fikset det

Reply:
IntelliJ klager hvis man ikke legger med en core import av typen
`@import (reference) '~nav-frontend-core/less/core'` hvis man bruker farger eller en mixin. Så tenkte det var lurt å alltid legge dette med

Reply:
Stemmer at vi hadde samme regel flere ganger i Chrome ja. Og jo mer jeg tenker på det så tror det egentlig bare var ett problem ved utvikling vi måtte fikse i forken. Og det gjør jo ting enda mindre viktig. 😛

`core` med `reference` eller import av `_variables` direkte må nok være med ja. Spiller Inge. Rolle hvilken dere bruker. Førstnevnte er det som brukes mest i nav-frontend, men effekten av de er den samme.